### PR TITLE
[FSTORE-884] Flyingduck docs

### PR DESCRIPTION
**Pull request by tdoehmen**

### CODE REVIEW

File: docs/setup_installation/common/arrow_flight_duckdb.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+When the service is enabled, clients will automatically use it for the following operations:
+
+- reading Feature Groups
+- reading External Feature Groups (supported Storage Connectors: Snowflake, BigQuery)
+- reading Queries (containing Feature Groups and supported External Feature Groups)
+- reading Batch Inference Data
+- creating In-Memory Training Datasets
+- writing Training Datasets
+- reading Training Datasets
```
Comment (SirOibaf) [inline docs/setup_installation/common/arrow_flight_duckdb.md:17]: This is WIP right?

File: docs/setup_installation/common/arrow_flight_duckdb.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+For larger datasets, clients can still make use of the Spark/Hive backend by explicitly setting 
+`read_options={"use_hive": True}` or `write_options={"use_spark": True}`, respectively.
```
Comment (SirOibaf) [inline docs/setup_installation/common/arrow_flight_duckdb.md:20]: I think this section belogs to the feature view user guide.

File: docs/setup_installation/common/arrow_flight_duckdb.md (hunk 1 above)
Comment (tdoehmen) [reply docs/setup_installation/common/arrow_flight_duckdb.md:17]: writing training datasets is WIP (probably 3.4.0), reading works

File: docs/setup_installation/common/arrow_flight_duckdb.md (hunk 2 above)
Comment (tdoehmen) [reply docs/setup_installation/common/arrow_flight_duckdb.md:20]: Okay, do you think I should also add it to the read_options section in the hsfs API docs?

File: docs/setup_installation/common/arrow_flight_duckdb.md (hunk 1 above)
Comment (SirOibaf) [reply docs/setup_installation/common/arrow_flight_duckdb.md:17]: Ok but then let's remove the writing part for the moment.

File: docs/setup_installation/common/arrow_flight_duckdb.md (hunk 2 above)
Comment (SirOibaf) [reply docs/setup_installation/common/arrow_flight_duckdb.md:20]: Yeah that would be great.

Comment (tdoehmen) [reply docs/setup_installation/common/arrow_flight_duckdb.md]: s. https://github.com/logicalclocks/feature-store-api/pull/1075

**Comment by tdoehmen** [comment]
Shall I open a cherry-picked PR for 3.3. for this and the hsfs docs?

**Review by SirOibaf — APPROVED** [review]

**Comment by SirOibaf** [comment]
> Shall I open a cherry-picked PR for 3.3. for this and the hsfs docs?

No that's not needed as the 3.3 branch for the documentation has not been cut yet.

**Pull request merged**